resources/assets/js/components/dues/DuesRequiredInfo.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div class="row">
+    <div class="col-12">
+      <form id="DuesRequiredInfoForm" v-on:submit.prevent="submit">
+        <h3>GT Directory Info</h3>
+        <p>Information obtained via GT Single Sign-On. Update at <a href="https://passport.gatech.edu">Passport</a>.</p>
```

Comment:
Could be misleading to people thinking they can change their name or other information even though not all of it can be changed. Not sure how I'd re-word it though.

Reply:
I think the goal here is to communicate that it cannot be changed within Apiary